internal/uvm/network.go

```diff
@@ -182,13 +183,26 @@ func (uvm *UtilityVM) addNIC(id guid.GUID, endpoint *hns.HNSEndpoint) error {
 				requesttype.Add,
 				nil),
 		}
-		// Uncomment this once we have GuestRequest support for Linux
-		//} else {
-		//	request.GuestRequest = guestrequest.GuestRequest{
-		//		ResourceType: guestrequest.ResourceTypeNetwork,
-		//		RequestType:  requesttype.Add,
-		//		Settings:     endpoint,
-		//	}
+	} else {
+		// Verify this version of LCOW supports Network HotAdd
+		if uvm.isNetworkNamespaceSupported() {
```

Comment:
Should there be an else to this?

Reply:
Perhaps an error would be cleaner. Just merge this though and get it in first :)